docs/validation-rules.rst

```diff
@@ -606,6 +606,16 @@ A list of types can be used to allow different values:
 
 .. note::
 
+    While the ``type`` is not required to be set at all, it is not encouraged to
+    leave it unset especially when using more complex rules such as ``schema``.
+    If you decide you still don't want to set an explicit type, rules such as
+    ``schema`` are only applied to values where the rules can actually be used
+    (such as `'dict'` and `'list'`). Also, the the case of ``schema``, cerberus
```

Comment:
I updated this to make it clearer that ``type`` and ``dict`` refer to the cerberus type rules and not the explicit python classes.